components/I18nAddressFields.js

```diff
@@ -299,7 +299,7 @@ const I18nAddressFields = ({
           label={fieldLabel}
           info={fieldInfo}
           value={value?.[fieldName]}
-          required={required === false ? false : !Object.keys(data?.optionalLabels).includes(fieldName)}
+          required={required === false ? false : !Object.keys(data?.optionalLabels || {}).includes(fieldName)}
```

Comment:
If you use includes, make sure it's an object.

Should it be an array actually?